web-app-aspnet/README.md

```diff
@@ -42,19 +42,19 @@ Use the following settings for your app registration:
 | **Platform type**              | `Web`                                              | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
 | **Redirect URIs**              | `https://localhost:5001/signin-oidc`               | Required value for this sample. <br/> You can change that later in your own implementation.                 |
 | **Front-channel logout URL**   | `https://localhost:5001/signout-oidc`              | Required value for this sample. <br/> You can change that later in your own implementation.                 |
-| **Client secret**              | _Value shown in Azure portal_                      | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).                 |
+| **Client secret**              | _Value shown in Microsoft Entra admin center_                      | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).                 |
```

Comment:
Should we include a link here, since folks might not know where the admin center is located?